nservicebus/containers/index_content_core_[,5].partial.md

```diff
@@ -5,6 +5,7 @@ Note that the instance of `IBus` is scoped for the lifetime of the container. He
 
 ### Cleanup
 
-When using an external container normally the bus instance is not disposed of manually. If `IBus.Dispose()` is called that would indirectly trigger the container to dispose lifetime scope.
+When using an external container normally the bus instance is not disposed of automatically. The following dispose sequence should be applied if latest container adapter packages are used:
```

Comment:
@danielmarbach - 

After trying to write this in a meaningful way to the user, I failed. As @SimonCropp pointed out, there's more ambiguity to be addressed. For example, he mentioned that this wasn't back ported to all versions of Version 5. For example, while it was back ported to it was back-ported to NServiceBus Unity 6.2.x, it was not back-ported to NServiceBus Unity 6.1.x.

Given that, I think it's best to include a table of versions that support this change, For example:
```
When using an external container, the bus instance is not automatically disposed. In order to dispose the resources properly:
1. Dispose the bus by calling IBus.Dispose().
2. Dispose the external container with container.Dispose()

This new behavior was introduced in the versions of the container that depend on NServiceBus  Versions 6 and above. It was also back-ported to some container versions that depend on NServiceBus Version 5 as a patch release. The code change is applicable to all the versions listed in the table below:

[Insert table of containers and versions that support this change]
```

As a side note, @Particular/nservicebus-maintainers - trying to document this points out the flaw in couple of things. This should not have been back ported in the first place. If it was back-ported, it should've been back-ported to not just partial versions. I believe having a docs PR at the time of making this change would have highlighted this problem.